hack/save-logs-files.sh

```diff
@@ -0,0 +1,64 @@
+#!/bin/bash
+
+# Copyright 2023 IBM Corp.
+# SPDX-License-Identifier: Apache-2.0
+
+# if a uid is passed, another file will be created, containing only logs with the uid
+fybrik_application_uid=$1
+
+# variable for limiting to recent logs, e.g. 50s, 20m, 3h
+duration=1h
+
+# create a directory for saving recent logs
+dirname_logs=fybrik_logs_$(date +%Y-%m-%d_%H-%M)_last_$duration
+mkdir $dirname_logs
+mkdir $dirname_logs/by_pod
+
+# iterate over the pods in the fybrik namespaces
+for ns in $(kubectl get ns -o=jsonpath='{.items[*].metadata.name}'); do
```

Comment:
kubectl is usually installed in hack/tools/bin. If it is not in $PATH, the executable will not be found.